custom_components/myskoda/coordinator.py

```diff
@@ -31,6 +31,8 @@
     DEFAULT_FETCH_INTERVAL_IN_MINUTES,
 )
 
+from . import handle_aiohttp_error
```

Comment:
This "feels" 🙃 . Would it make sense to have this in an `error_handlers.py` instead?

Reply:
Yup. Will improve. Thanks! 👍